charts/istio-authorizer/values.yaml

```diff
@@ -185,6 +273,6 @@ data:
   # The data should match istio-authorizer configuration options
   # https://docs.authorization.cloudentity.com/guides/developer/protect/istio/istio_authz_config/
   #
-  config: |
+  extraconfig: |
```

Comment:
the config should be an object, not a string

Reply:
fixed